rpc/core/types/responses.go

```diff
@@ -145,10 +144,8 @@ type ResultDialPeers struct {
 
 // A peer
 type Peer struct {
-	NodeInfo         types.NodeInfo       `json:"node_info"`
-	IsOutbound       bool                 `json:"is_outbound"`
-	ConnectionStatus p2p.ConnectionStatus `json:"connection_status"`
-	RemoteIP         string               `json:"remote_ip"`
+	ID  types.NodeID `json:"node_id"`
```

Comment:
Looks like we can still get a status field form the [peer manager](https://github.com/tendermint/tendermint/blob/18784f7fe0fa316994dd94cbd84573d87d7958d4/internal/p2p/peermanager.go#L966), any reason to not pull this in as well?

Reply:
that `PeerStatus` isn't particularly useful in this context.